### PR TITLE
Refactored OneAuthTestApp infra in order to process LTW test cases

### PR DESCRIPTION
**Pull request by amoyixi**
This pr is about refactoring OneAuthTestApp infra in order to process LTW test cases
1. add method `handlePreferBrokerSwitchButton`
2. add method `selectFromAppConfiguration`
3. add method `handleSignInWithoutPrompt`

### CODE REVIEW

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/OneAuthTestApp.java
Comment (fadidurah) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/OneAuthTestApp.java:1]: Seems handleUserNameInput was not added

**Review by fadidurah — APPROVED** [review]

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/OneAuthTestApp.java [hunk 1]
```diff
@@ -141,6 +141,11 @@ private void signIn(@NonNull final String username,
         new AadPromptHandler(promptHandlerParameters).handlePrompt(username, password);
     }
 
+    public String handleSignInWithoutPrompt() {
+        UiAutomatorUtils.handleButtonClick("com.microsoft.oneauth.testapp:id/sign_in_button");
+        return getTokenSecret();
+    }
+
```
Comment (somalaya) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/OneAuthTestApp.java]: can you confirm if the indentation here is right? It seems off to me

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/OneAuthTestApp.java [hunk 2]
```diff
@@ -141,6 +141,11 @@ private void signIn(@NonNull final String username,
         new AadPromptHandler(promptHandlerParameters).handlePrompt(username, password);
     }
 
+    public String handleSignInWithoutPrompt() {
```
Comment (somalaya) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/OneAuthTestApp.java]: your method name says sign in without prompt. But you are returning a secret. These look a bit disconnected. I would suggest you move your assert statements to this method. Assert here itself if the token is valid. then you do not have to return the token to the test

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/OneAuthTestApp.java
Comment (amoyixi) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/OneAuthTestApp.java:1]: My mistake, handleUserNameInput already got approved in commit #2103. Modified pr description by removing handleUserNameInput.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/OneAuthTestApp.java (hunk 1 above)
Comment (amoyixi) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/OneAuthTestApp.java]: I've checked and the indentation is correct.

**Review by somalaya — APPROVED** [review]

**Pull request merged**